aladdin/lib/k8s/helm.py

```diff
@@ -107,6 +107,24 @@ def find_values(self, chart_path, cluster_name, namespace):
             logger.info("Found site values file")
             values.append(site_values_path)
 
+        cluster_config_values_path = os.path.join(
+            os.environ["ALADDIN_CONFIG_DIR"],
+            cluster_name,
+            "values.yaml"
+        )
+        if os.path.isfile(cluster_config_values_path):
+            logger.info("Found cluster config values file")
+            values.append(cluster_config_values_path)
+        cluster_namespace_config_values_path = os.path.join(
+            os.environ["ALADDIN_CONFIG_DIR"],
+            cluster_name,
+            namespace,
+            "values.yaml"
+        )
+        if os.path.isfile(cluster_namespace_config_values_path):
+            logger.info("Found cluster namespace config values file")
+            values.append(cluster_namespace_config_values_path)
+
```

Comment:
I think order here is important, so maybe these should be added to the values list before any project configuration?

Reply:
let's also define a local consistency here where we define the file at time of use for each `if` block, or we define them all at once at the top

Reply:
I've taken order into account 👍 I've added a comment to make it explicit/clear. I also moved the variable definition close to the `if` block.